refactor(web): Restructure the operation buttons layout in the app information component (#21742)

HyaCiovo opened the pull request:
> [!IMPORTANT]
>
> 1. Make sure you have read our [contribution guidelines](https://github.com/langgenius/dify/blob/main/CONTRIBUTING.md)
> 2. Ensure there is an associated issue and you have been assigned to it
> 3. Use the correct syntax to link this PR: `Fixes #<issue number>`.

## Summary

- Display the import DSL and switch app buttons directly on the interface, no longer hiding them in the more options menu

Fixes #21742 

Since the "more" button only hides a single option, and considering that in languages with longer words the buttons would still wrap even with the "more" option, it's better to eliminate the "more" button altogether and display all buttons directly on the interface.
![87bebbaffba80e3e3247d7035a4f5ca](https://github.com/user-attachments/assets/dc2678a7-feeb-4462-bf6d-ad688c706940)


## Screenshots

| Before | After |
|--------|-------|
| ![image](https://github.com/user-attachments/assets/6d4a65f6-7ce4-47c4-b139-9c7fb8c3bf50)    | ![image](https://github.com/user-attachments/assets/49b3951a-46e6-409f-a502-5bdf60dfa453)  |

## Checklist

- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [x] I understand that this PR may be closed in case there was no previous discussion or issues. (This doesn't apply to typos!)
- [x] I've added a test for each change that was introduced, and I tried as much as possible to make a single atomic change.
- [x] I've updated the documentation accordingly.
- [x] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

HyaCiovo committed:
refactor(web): Restructure the operation buttons layout in the app information component (#21742)

- Display the import DSL and switch app buttons directly on the interface, no longer hiding them in the more options menu

crazywoola reviewed (changes requested):
![Image](https://github.com/user-attachments/assets/87bf46e9-c2b9-4458-aff5-c5be2d0a99fb)

If the button `more` can not displayed in a row, the button `more` should contain 2 buttons.

![Image](https://github.com/user-attachments/assets/fc6ee0f5-c751-420f-bf2b-5e654504f5d7)

![Image](https://github.com/user-attachments/assets/8279a7c3-6812-4faa-b75f-300667b95b44)

HyaCiovo committed:
refactor(app-sidebar): Reconstruct application operation button layout

- Move application operation buttons to a separate component AppOperations
- Optimize the display logic of operation buttons, add more buttons to adapt to different screen widths
- Use the PortalToFollowElem component to implement the pop-up menu for additional operations
- Use useEffect to dynamically calculate visible operations and additional operations, improving responsive design
